packages/application-extension/src/index.tsx

```diff
@@ -328,6 +331,32 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isEnabled: () => !labShell.isEmpty('right')
       });
 
+      commands.addCommand(CommandIDs.activateSidebarWidget, {
+        label: trans.__('Open Sidebar Element'),
+        caption: trans.__('By default opens first element of left sidebar'),
+        execute: args => {
+          const index = parseInt(args.index as string, 10);
+          if (args.side != 'left' && args.side != 'right') {
+            throw Error(`Unsupported sidebar: ${args.side}`);
+          }
+          const widgets = Array.from(labShell.widgets(args.side));
+          if (index >= widgets.length) {
+            throw Error(`No item with index ${index} in ${args.side} sidebar`);
```

Comment:
```suggestion
            return;
```
Maybe it can do nothing silently.